src/oom/oomd-conf.c

```diff
@@ -75,6 +75,7 @@ void manager_parse_config_file(Manager *m) {
                 { "OOM", "SwapUsedLimit",                    config_parse_permyriad, 0, &m->swap_used_limit_permyriad          },
                 { "OOM", "DefaultMemoryPressureLimit",       config_parse_loadavg,   0, &m->default_mem_pressure_limit         },
                 { "OOM", "DefaultMemoryPressureDurationSec", config_parse_duration,  0, &m->default_mem_pressure_duration_usec },
+                { "OOM", "PrekillHookDeadlineSec",           config_parse_sec,       0, &m->prekill_deadline                   },
```

Comment:
Deadline is usually a timestamp in the future when an operation should stop. In your case this seems to be a timeout. You can use timeout to calculate deadline (ideally with monotonic clocks).